src/xenia/base/math.h

```diff
@@ -330,7 +330,7 @@ inline T rotate_right(T v, uint8_t sh) {
   constexpr unsigned char SHIFT_MASK = (CHAR_BIT * sizeof(T)) - 1;
   uint8_t rshr = sh & SHIFT_MASK;
   uint8_t lshl = static_cast<uint8_t>(-static_cast<int8_t>(sh)) & SHIFT_MASK;
-  return (n >> rshr) | (n << lshl);
+  return (v >> rshr) | (v << lshl);
```

Comment:
What does this have to do with C++20?

Reply:
With `n` with C++20:
![image](https://github.com/user-attachments/assets/b8940921-8e0b-4f37-91bf-68d114af2d9c)

With `v` with C++20:
Just no errors